src/inmanta/loader.py

```diff
@@ -500,9 +443,6 @@ def convert_module_to_relative_path(full_mod_name: str) -> str:
 
     module_parts.insert(1, PLUGIN_DIR)
 
-    if module_parts[-1] == "__init__":
-        module_parts = module_parts[:-1]
-
```

Comment:
What's this about?

Reply:
Nice catch, I think this is an artifact from an earlier implementation.